notifications/07-13-21-1500.md

```diff
@@ -0,0 +1,32 @@
+
+**What is happening?**
+
+CHG0027891 - CITZ - SILVER OCP4 - Migrate additional INFRA VMs to newer ESXi cluster
+
+As part of ongoing maintanance for VMWare-managed infrastructure, we need to migrate three VMs responsible for giving the Openshift 4 SILVER cluster more capacity for its ingress router pods over to a newer ESXi cluster.
+
+This migration will take place outside of business hours to ensure reduced impact for open connections as each VM is migrated in turn.
+
+
+**When?**
+
+Starting Thursday July 15th 2021 starting at 6pm PDT and will take a maximum of one hour.
```

Comment:
Maybe we don't need "starting" twice in one sentence?

```suggestion
Starting Thursday July 15th 2021 at 6pm PDT and will take a maximum of one hour.
```

Reply:
thought I dealt with that one